testenv/bootbox/puppet/modules/genesis/templates/config.yaml.erb

```diff
@@ -19,6 +19,7 @@
         :gpgcheck:  0
 :gem_files:
     "genesis_framework": http://<%= @genesis_ipaddress %>:8888/gem/genesis_framework
+:gem_args: &gem_args --no-ri --no-rdoc --clear-sources # Arguments to pass to gem install. Can include --source for internal repos
```

Comment:
I don't think you should include --clear-sources here since without a --source the gem install will probably fail.  So either:
:gem_args: &gem_args --no-ri --no-rdoc
or
:gem_args: &gem_args --no-ri --no-rdoc --clear-sources --source https://gems.example.com